addons/website/static/src/snippets/s_dynamic_snippet/dynamic_snippet.js

```diff
@@ -91,7 +91,7 @@ export class DynamicSnippet extends Interaction {
                     "filter_id": parseInt(nodeData.filterId),
                     "template_key": nodeData.templateKey,
                     "limit": parseInt(nodeData.numberOfRecords),
-                    "search_domain": this.getSearchDomain(),
+                    "search_domain": await this.getSearchDomain(),
```

Comment:
Hi @jeh-odoo ,

I may need some explanation here :p we do not really make the method async right? (PR title / Commit message)

Actually I am surprised as I do not see `async getSearchDomain()` in the code base.

Maybe this is something linked to new Interactions framework?

I feel like I am missing something :p 

Would it work to instead assign `const searchDomain = this.getSearchDomain();` below `nodeData` and use it in directly in the object assignation?

Thanks for the help & Cheers!

Reply:
Hello there,
In fact, I made the function async but in the enterprise PR (to get the country info) but indeed the commit was a bit misleading here.
I've updated a bit the message to be more precise.

Reply:
Hi again @jeh-odoo 

LGTM overall. I would add link to COM PR in ENT PR and vice versa. Also, we agree that this is an IMP (we did not filter on country before in the snippet) ?

The only question I had was the fact that we use await on a method that is not always async, but the doc on 'await' seems to be ok with it as it will be converted to a resolved promise and awaited :)

So, once you're done with the small PR / commit message enhancements, you can green. Cheers!

Reply:
(and maybe rebase, I notice branches are a bit old)